src/media/headset.ts

```diff
@@ -0,0 +1,71 @@
+import { Observable } from 'rxjs';
+import { SdkMedia } from "..";
+import HeadsetService, { ConsumedHeadsetEvents, VendorImplementation} from 'softphone-vendor-headsets';
+
+export class SdkHeadset {
+    private media: SdkMedia;
+    private headsetLibrary: HeadsetService;
+    headsetEvents: Observable<ConsumedHeadsetEvents>;
+
+    constructor(media) {
+        this.media = media;
+        this.headsetLibrary = HeadsetService.getInstance({ logger: console });
+        this.headsetEvents = this.headsetLibrary.headsetEvents$;
+    }
+
+    getAudioDevice(newMicId: string): void {
```

Comment:
It's a bit unconventional to have a `get*` fn set something a return nothing. We should change this to something more appropriate like `updateInputDevice` or `changeInputDevice` or something along those lines.